Modules/Core/Common/test/CMakeLists.txt

```diff
@@ -555,9 +555,9 @@ if("${CMAKE_SYSTEM_NAME}" MATCHES "Darwin|Linux")
   itk_add_test(NAME itkFloatingPointExceptionsTest2 COMMAND ITKCommon1TestDriver
     itkFloatingPointExceptionsTest ZeroDivByZero)
   itk_add_test(NAME itkFloatingPointExceptionsTest3 COMMAND ITKCommon1TestDriver
-    itkFloatingPointExceptionsTest FPOverFlow)
+    itkFloatingPointExceptionsTest FPOverflow)
   itk_add_test(NAME itkFloatingPointExceptionsTest4 COMMAND ITKCommon1TestDriver
-    itkFloatingPointExceptionsTest FPUnderFlow)
+    itkFloatingPointExceptionsTest FPUnderflow)
```

Comment:
Why not just "Overflow" and "Underflow"? The prefix "FP" does not seem necessary, and it isn't used for "DivByZero" and "ZeroDivByZero" either.

Reply:
These tests are part of Common module, which has a lot of tests. FP prefix to the tests names is useful, at least to me.

Reply:
I'll leave it as is.

Reply:
Merged 👍  I guess the "FP" prefix stands for _floating point_, right? If so, then the prefix seems unnecessary, as the names of the tests `itkFloatingPointExceptionsTest3` and `itkFloatingPointExceptionsTest4` already make it clear to me that they are about floating points. Anyway, it's unrelated to the capital "F" in Over/UnderFlow, so I rest my case for now  😺

Reply:
> itkFloatingPointExceptionsTest4

I didn't see this name 😄 I thought that FPUnderflow was test name.